gvr-javascript/app/build.gradle

```diff
@@ -25,4 +25,9 @@ android {
         jni.srcDirs = [] // no auto generation of Android.mk
         jniLibs.srcDir 'libs' // pre-compiled libraries
     }
+}
+
+dependencies {

```

Comment:
Please update to
```
dependencies {
    if (findProject(':script')) {
        implementation project (':script')
    } else {
        debugCompile(name: 'script-debug', ext: 'aar')
        releaseCompile(name: 'script-release', ext: 'aar')
    }
}
```

Makes it easier to build the app together with the framework.

Reply:
Will do